zer0-os/zOS
# feat: update invite screen and layout

### What does this do?
- updates invite screen to follow latest designs

### Why are we making this change?
- as per figma designs

### How do I test this?
- visit zOS `/get-access` page. Check against designs [here](https://www.figma.com/file/aETpyuG2AKjNcQtCjldcX0/ZERO-Messenger-%2F-Web?type=design&node-id=9745-269425&mode=design&t=tEwW07rdhHz6gJWy-4)

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?



<img width="1412" alt="Screenshot 2023-07-27 at 11 44 04" src="https://github.com/zer0-os/zOS/assets/39112648/64dd5e33-c1a2-46b3-a268-a12ae49b3b12">

<img width="1412" alt="Screenshot 2023-07-27 at 11 44 09" src="https://github.com/zer0-os/zOS/assets/39112648/7a4aa456-95ed-4596-b84a-6799fb72d66e">

<img width="1412" alt="Screenshot 2023-07-27 at 11 50 08" src="https://github.com/zer0-os/zOS/assets/39112648/540cc449-cf63-48a2-af94-b2f072af9520">
